src/datachain/lib/dc.py

```diff
@@ -1934,7 +1934,7 @@ def from_csv(
         session: Optional[Session] = None,
         settings: Optional[dict] = None,
         column_types: Optional[dict[str, "Union[str, ArrowDataType]"]] = None,
-        newlines_in_values: bool = False,
+        parse_options_config: Optional[dict[str, "Union[str, bool]"]] = None,
```

Comment:
(minor, just to brainstorm a bit, feel free to ignore)

How about: `parse_options` or `parse_config` - to make it less verbose and smaller in size (client code will look better a bit potentially)